### DISCUSSION TITLE
Can't we use PostgreSQL for PDS any longer?

### DISCUSSION THREAD

**Discussion by YOCKOW** [bluesky-social/atproto#2287] [How to do it?] [answered]
[PR#1705](https://github.com/bluesky-social/atproto/pull/1705) had removed `PDS_DB_POSTGRES_*` environment variables from [packages/pds/src/config/env.ts](https://github.com/bluesky-social/atproto/pull/1705/files#diff-7cef43e80332e39496301090c631bb829c61afe6e0bf14438e4ef35ec1755457).

Does that mean we can't use PostgreSQL for PDS any longer?
Or are there any chance to use PostgreSQL?

**Comment by devinivy** [comment, marked as answer]
Yeah, that's correct—our PDS distribution only supports sqlite.  It is designed as a single-tenant architecture, so each repository is contained within its own sqlite database.

**Reply by YOCKOW** [reply]
Thank you for the answer.
I'm very sad, but it is what it is.
